### PR TITLE
[docs] PR for Cosmetic changes to benchmark/tpcc-ysql files (Doc9487)

### PR DESCRIPTION
**Pull request by lizayugabyte**
Edited files for clarity and consistency

### CODE REVIEW

File: docs/content/latest/benchmark/tpcc-ysql/10-wh/tpcc-ysql.md [hunk 1]
```diff
@@ -82,7 +78,7 @@ $ ./tpccbenchmark --execute=true \
   </tbody>
 </table>
 
-Once the execution is done the TPM-C number along with the efficiency is printed.
+Once the execution is completed, the TPM-C number along with the efficiency is printed, as follows:
 
 ```
 21:09:23,588 (DBWorkload.java:955) INFO  - Throughput: Results(nanoSeconds=1800000263504, measuredRequests=8554) = 4.752221526539232 requests/sec reqs/sec
```
Comment (ddhodge) [inline docs/content/latest/benchmark/tpcc-ysql/10-wh/tpcc-ysql.md:84]:
```output?
there's an extra space at the bottom before the closing ``` which creates an empty code block

File: docs/content/latest/benchmark/tpcc-ysql/100-wh/tpcc-ysql.md [hunk 2]
```diff
@@ -83,7 +80,7 @@ $ ./tpccbenchmark --execute=true \
   </tbody>
 </table>
 
-Once the execution is done the TPM-C number along with the efficiency is printed.
+Once the execution is completed, the TPM-C number along with the efficiency is printed, as follows:
 
 ```
 04:54:54,560 (DBWorkload.java:955) INFO  - Throughput: Results(nanoSeconds=1800000866600, measuredRequests=85196) = 47.33108832382159 requests/sec reqs/sec
```
Comment (ddhodge) [inline docs/content/latest/benchmark/tpcc-ysql/100-wh/tpcc-ysql.md:86]:
```output?

File: docs/content/latest/benchmark/tpcc-ysql/1000-wh/tpcc-ysql.md [hunk 3]
```diff
@@ -84,7 +81,7 @@ $ ./tpccbenchmark --execute=true \
 </table>
 
 
-Once the execution is done the TPM-C number along with the efficiency is printed.
+Once the execution is completed, the TPM-C number along with the efficiency is printed, as follows:
 
 ```
 17:18:58,728 (DBWorkload.java:955) INFO  - Throughput: Results(nanoSeconds=1800000716759, measuredRequests=842216) = 467.8975914612168 requests/sec reqs/sec
```
Comment (ddhodge) [inline docs/content/latest/benchmark/tpcc-ysql/1000-wh/tpcc-ysql.md:87]:
```output?

File: docs/content/latest/benchmark/tpcc-ysql/100-wh/tpcc-ysql.md [hunk 4]
```diff
@@ -1,9 +1,6 @@
-### TPC-C Load Phase
+## TPC-C Load Phase
 
-Before starting the workload, you will need to load the data first. Make sure
-to replace the IP addresses with that of the nodes in the cluster. Loader
-threads allow us to configure the number of threads used to load the data. For
-a 3 node c5d.4xlarge cluster, loader threads value of 48 was the most optimal.
+Before starting the workload, you need to load the data. Make sure to replace the IP addresses with that of the nodes in the cluster. Loader threads allow you to configure the number of threads used to load the data. For a 3-node c5d.4xlarge cluster, loader threads value of 48 was the most optimal.
```
Comment (ddhodge) [inline docs/content/latest/benchmark/tpcc-ysql/100-wh/tpcc-ysql.md:4]: a loader threads value of 48 was optimal.

File: docs/content/latest/benchmark/tpcc-ysql/1000-wh/tpcc-ysql.md [hunk 5]
```diff
@@ -1,9 +1,6 @@
-### TPC-C Load Phase
+## TPC-C Load Phase
 
-Before starting the workload, you will need to load the data first. Make sure
-to replace the IP addresses with that of the nodes in the cluster. Loader
-threads allow us to configure the number of threads used to load the data. For
-a 3 node c5d.4xlarge cluster, loader threads value of 48 was the most optimal.
+Before starting the workload, you need to load the data first. Make sure to replace the IP addresses with that of the nodes in the cluster. Loader threads allow you to configure the number of threads used to load the data. For a 3-node c5d.4xlarge cluster, loader threads value of 48 was the most optimal.
```
Comment (ddhodge) [inline docs/content/latest/benchmark/tpcc-ysql/1000-wh/tpcc-ysql.md:4]: a loader threads value of 48 was optimal.

**Review by ddhodge — APPROVED** [review]
Few minor comments, 1 fix needed

File: docs/content/latest/benchmark/tpcc-ysql/10-wh/tpcc-ysql.md (hunk 1 above)
Comment (lizayugabyte) [reply docs/content/latest/benchmark/tpcc-ysql/10-wh/tpcc-ysql.md:84]: Fixed

File: docs/content/latest/benchmark/tpcc-ysql/100-wh/tpcc-ysql.md (hunk 2 above)
Comment (lizayugabyte) [reply docs/content/latest/benchmark/tpcc-ysql/100-wh/tpcc-ysql.md:86]: Fixed

File: docs/content/latest/benchmark/tpcc-ysql/1000-wh/tpcc-ysql.md (hunk 3 above)
Comment (lizayugabyte) [reply docs/content/latest/benchmark/tpcc-ysql/1000-wh/tpcc-ysql.md:87]: Fixed

File: docs/content/latest/benchmark/tpcc-ysql/1000-wh/tpcc-ysql.md (hunk 5 above)
Comment (lizayugabyte) [reply docs/content/latest/benchmark/tpcc-ysql/1000-wh/tpcc-ysql.md:4]: Fixed

File: docs/content/latest/benchmark/tpcc-ysql/100-wh/tpcc-ysql.md (hunk 4 above)
Comment (lizayugabyte) [reply docs/content/latest/benchmark/tpcc-ysql/100-wh/tpcc-ysql.md:4]: Fixed

**Pull request merged**